include/aws/common/common.h

```diff
@@ -117,12 +117,12 @@
  * Violations of the function contracts are undefined behaviour.
  */
 #ifdef CBMC
-#define AWS_PRECONDITION(cond) __CPROVER_assume(cond)
+#define AWS_PRECONDITION(cond, explanation) __CPROVER_precondition((cond), (explanation))
```

Comment:
Might make sense to append "Precondition: " to the string?

Reply:
I agree :)